superset/errors.py

```diff
@@ -156,6 +157,9 @@ class SupersetErrorType(str, Enum):
     SupersetErrorType.CONNECTION_PORT_CLOSED_ERROR: [
         {"code": 1008, "message": _("Issue 1008 - The port is closed.")},
     ],
+    SupersetErrorType.CONNECTION_INVALID_PORT_ERROR: [
+        {"code": 1021, "message": _("Issue 1021 - Port number is invalid.")},
```

Comment:
Issue 1021 is taken, we need to create a new issue here.

Reply:
i'm working on this now